cql2/standard/schema/cql2.json

```diff
@@ -250,22 +250,22 @@
           "type": "string",
           "enum": ["a_containedBy", "a_contains", "a_equals", "a_overlaps"]
         },
-        "args": {"$ref": "#/$defs/arrayExpression"}
+        "args": {"$ref": "#/$defs/arrayOperand"}
       }
     },
-    "arrayExpression": {
+    "arrayOperand": {
```

Comment:
@pvretano I think this should be `arrayOperands` plural to be consistent with the other rules and to highlight the fact that this represents two operands for the arra predicate functions.

Reply:
I agree, this should be plural. This applies also to cql2.yaml.